single-campaigns.php

```diff
@@ -1,46 +1,108 @@
 <?php
 /**
- * The Template for displaying all single campaign posts
+ * Template Variables for Campaigns.
  *
- * Methods for TimberHelper can be found in the /lib sub-directory
- *
- * @package  WordPress
- * @subpackage  Timber
- * @since    Timber 0.1
+ * @package P4MT
  */
 
 use Timber\Timber;
 
 // Initializing variables.
 $context = Timber::get_context();
 
-// @var P4_Post object $post
+/**
+ * Post object.
+ *
+ * @var P4_Post $post
+ * */
 $post            = Timber::query_post( false, 'P4_Post' );
 $context['post'] = $post;
 
+// Save custom style settings.
+$custom_styles = [];
+
+$custom_styles['css']['nav_color']     = $post->campaign_nav_color ? ".navbar { background-color: {$post->campaign_nav_color} !important;}" : null;
```

Comment:
I don't see any custom fields set for the custom post type. I guess this would be part of another PR?

Reply:
It was part of another branch, but I've merged into this one. I'm assuming you mean the metabox fields?